howto-conduct-a-privacy-review.md

```diff
@@ -113,8 +113,13 @@ of a privacy review, or might even file contradictory privacy issues if
 PING members disagree. Disagreement is rare. In the case that a PING privacy review results in 
 contradictory (or mutually exclusive) issues being filed, its the responsibility
 of the working group making the proposal (and, ultimately the director)
-to figure out how to proceed. However, the chairs will work with the working group and reviewer as needed with the aim of resolving any outstanding issues. 
+to figure out how to proceed. However, the chairs will work with the working group and reviewer
+as needed with the aim of resolving any outstanding issues.
 
+If a reviewer is unable to resolve an issue with the working group
+and the issue may lead to a formal objection, the reviewer
+is encouraged to disucss the issue with the PING chairs, and on a PING call.
+PING can then explore other ways to try and resolve the concern without the formal objection.
```

Comment:
An additional goal for PING would be to document the concern and positions of PING reviewers and the Working Group, to facilitate subsequent resolutions of any Formal Objection.

Reply:
thats a good change, thanks! done